invenio_app_ils/eitems/mappings/v6/eitems/eitem-v1.0.0.json

```diff
@@ -48,16 +48,31 @@
                       "type": "object"
                     },
                     "name": {
-                      "type": "keyword"
+                      "type": "text",
+                      "fields": {
+                        "keyword": {
+                          "type": "keyword"
+                        }
+                      }
                     }
                   },
                   "type": "object"
                 },
                 "alternative_names": {
-                  "type": "keyword"
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword"
+                    }
+                  }
                 },
                 "full_name": {
-                  "type": "text"
+                  "type": "text",
```

Comment:
We can keep it like this, but in general this is useful when we want to create a search filter on a text field (you need a `keyword` type). I am not sure we will do such a thing for `names` in general...

Reply:
Yes, apart from the search filter, it's also usefull for efficiently search for the full name